api_spec.yaml

```diff
@@ -661,44 +663,40 @@ paths:
                                 type: string
                                 enum:
                                     - Not Found
-    /user/email/verify:
-        get:
+    /<tenantId>/dashboard/api/user/email/verify:
+        put:
             tags:
```

Comment:
there is a a syntax error somewhere. Copy / paste this into https://editor.swagger.io/  and see.

Reply:
updated